runtime/Cargo.toml

```diff
@@ -7,7 +7,7 @@ name = "encointer-node-notee-runtime"
 repository = "https://github.com/encointer/encointer-node/"
 # minor revision must match node/client
 # patch revision must match runtime spec_version
-version = "1.4.25"
+version = "1.4.26"
```

Comment:
I do think that this is at least a minor version bump.

Reply:
agreed